### ISSUE TITLE
Produces invalid SBOM file when CVSSv31 method is used in input

### ISSUE THREAD

**Issue by lazka** [CycloneDX/cyclonedx-cli#359]
Input file: [buginput.json](https://github.com/CycloneDX/cyclonedx-cli/files/14758685/buginput.json)

Passing the attached SBOM through cyclonedx-cli results in an invalid JSON:

```console
$ cyclonedx --version                                                                
0.25.0
$ # it's valid
$ cyclonedx validate --input-file buginput.json
BOM validated successfully.
```

```console
$ # converting breaks it
$ cyclonedx convert --input-file buginput.json --output-file converted.json
$ cyclonedx validate --input-file converted.json
Unable to validate against any JSON schemas.
BOM is not valid.
```

```console
$ # merging breaks it too
$ cyclonedx merge --input-files buginput.json buginput.json --output-file merged.json
$ cyclonedx validate --input-file merged.json   
Unable to validate against any JSON schemas.
BOM is not valid.
```

from what I see the rating "method" gets encoded wrongly:

```json
{
  "score": 3.3,
  "severity": "low",
  "method": "CVSSv31",
  "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:L/A:N"
}
```

After:

```json
{
  "score": 3.3,
  "severity": "low",
  "method": "CVSSv3.1",
  "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:L/A:N"
}
```

**Comment by andreas-hilti** [comment]
Is this related to https://github.com/CycloneDX/cyclonedx-dotnet-library/pull/261?

**Comment by mtsfoni** [comment]
Should be fixed in 0.25.1

**Comment by lazka** [comment]
Confirmed. Thanks!
